packages/xs-vat-worker/xs-lin.mk

```diff
@@ -20,14 +33,19 @@ moddable/xs/platforms/lin_xs_cli.c: moddable/xs/platforms/lin_xs.h
 
 moddable/xs/platforms/lin_xs.h: /usr/include/glib-2.0/gio/gio.h
 
+.PHONY: install-deps
+install-deps:
+	$(MAKE) -f xs-lin.mk install-gio
+
 .PHONY: install-gio
 /usr/include/glib-2.0/gio/gio.h:
-	@echo "GIO not installed, need root to run apt-get install libgio2.0-dev
+	@echo "GIO not installed, need root to run apt-get install libglib2.0-dev"
 	@echo "feel free to stop now and run `make install-gio` directly, then rebuild"
-	$(MAKE) install-gio
+	$(MAKE) -f xs-lin.mk  install-gio
 
+# ubuntu 'bionic' puts gio in libglib2.0-dev
 install-gio:
-	sudo apt-get -y update && sudo apt-get -y install libgio2.0-dev
+	sudo apt-get -y update && sudo apt-get -y install libglib2.0-dev
```

Comment:
I'm surprised that suffices... I think it won't if/when we use sockets from within XS. But I'm OK with it for now.

Reply:
From what I can tell (https://packages.ubuntu.com/search?keywords=libgio&searchon=names&suite=all&section=all) `libgio2.0-dev` isn't actually a package. I think `libgio` and `libglib` header/`.a` files are both included in the `libglib2.0-dev` package.